### DISCUSSION TITLE
Command line arguments parser

### DISCUSSION THREAD

**Discussion by BFH-ktt1** [logisim-evolution/logisim-evolution#980] [Dev talk]
Currently the command-line parser in "Startup.java" is really "a mess" and not intelligent. It would be nice to modernize this parser such that it would also allow for long parameters like "--version" and their short ones "-v" (as proposed by @R3dstOne in #963).
Any ideas and any takers?

**Comment by MarcinOrlowski** [comment]
What about using i.e. https://commons.apache.org/proper/commons-cli/

**Reply by R3dst0ne** [reply]
This library looks fine. It supports both POSIX and the GNU extension (long options).
I think we should conform tho those two standards.

**Comment by BFH-ktt1** [comment]
Also fine by me

**Comment by MarcinOrlowski** [comment]
OK, implementation progress assigned to this ticket -> #981
